docs/content/how-to/openstack/performance-tuning.md

```diff
@@ -0,0 +1,143 @@
+  informations so the instance will be scheduled on a host with capable hardware (e.g. Intel or Mellanox NICs capable
+  of SR-IOV, DPDK, etc).
+- An OpenStack network attached to the SR-IOV or DPDK capable NICs; this network must be usable by the project
+  used by the HostedCluster.
+
+## Node tunning
+
+Before creating an additional Nodepool used for performance workloads, we'll create a PerformanceProfile that will be
+used for the nodes.
+
+The PerformanceProfile is a custom resource that allows you to define the performance characteristics of a node. The
+PerformanceProfile can be used to configure the CPU, memory, hugepages, real-time kernel, and other performance-related
+settings of a node.
+
+Here is an example of a PerformanceProfile that can be used to configure a node for high-performance workloads:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: perfprof-1
+  namespace: clusters
+data:
+  tuning: |
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: cnf-performanceprofile
+      namespace: "${HYPERSHIFT_NAMESPACE}"
+    data:
+      tuning: |
+        apiVersion: performance.openshift.io/v2
+        kind: PerformanceProfile
+        metadata:
+          name: cnf-performanceprofile
+        spec:
+          additionalKernelArgs:
+            - nmi_watchdog=0
+            - audit=0
+            - mce=off
+            - processor.max_cstate=1
+            - idle=poll
+            - intel_idle.max_cstate=0
+            - amd_iommu=on
+          cpu:
+            isolated: "${CPU_ISOLATED}"
+            reserved: "${CPU_RESERVED}"
+          hugepages:
+            defaultHugepagesSize: "1G"
+            pages:
+              - count: ${HUGEPAGES}
+                node: 0
+                size: 1G
+          nodeSelector:
+            node-role.kubernetes.io/worker: ''
+          realTimeKernel:
+            enabled: false
+          globallyDisableIrqLoadBalancing: true
+```
+
+Create this PerformanceProfile (change the namespace field in the example above to specify the clusters namespace) by running:
+
+```shell
+oc apply -f perfprof.yaml
+```
+
+## Nodepool creation
+
+After creating the PerformanceProfile, we can create a Nodepool that will use the PerformanceProfile for the nodes.
+
+```shell
+export NODEPOOL_NAME=$CLUSTER_NAME-cnf
+export IMAGE_NAME="rhcos"
+export FLAVOR="m1.xlarge.nfv"
+
+hcp create nodepool openstack \
+  --cluster-name $CLUSTER_NAME \
+  --name $NODEPOOL_NAME \
+  --node-count 0 \
+  --openstack-node-image-name $IMAGE_NAME \
+  --openstack-node-flavor $FLAVOR
+```
+
+Now let's patch the Nodepool to use the PerformanceProfile:
+
+```shell
+oc patch nodepool -n ${HYPERSHIFT_NAMESPACE} ${CLUSTER_NAME} -p '{"spec":{"tuningConfig":[{"name":"cnf-performanceprofile"}]}}' --type=merge
+```
+
+We can scale the Nodepool to the desired number of nodes:
+
+```shell
+oc scale NodePool/$CLUSTER_NAME --namespace ${HYPERSHIFT_NAMESPACE} --replicas=1
```

Comment:
To not lose @stephenfin's comment, he also suggested the customer would use `--render-sensitive` here instead of patching an existing object after its creation. I liked the idea of patching because I think it's less error prone, but I'll listen for feedback here and adjust.

Reply:
```
Rather than doing this, would now be a good time to introduce the --render and --render-sensitive options?
```
His comment ^